packages/fcl-wc/src/service.js

```diff
@@ -0,0 +1,188 @@
+import QRCodeModal from "@walletconnect/qrcode-modal"
+import {config} from "@onflow/config"
+import {invariant} from "@onflow/util-invariant"
+import {isMobile} from "./utils"
+
+export const makeServicePlugin = client => ({
+  name: "fcl-service-walletconnect",
```

Comment:
@JeffreyDoyle Any thoughts on spec of this?
Should we use f_type perhaps for the plugin type?
Anything else you think is missing?

Reply:
Could try adding `f_type: "ServicePlugin"`. `f_type` is there to declare the type of data structure, so in this case it's a "ServicePlugin" that is type "discovery-service"